src/components/global/Playground/device-preview.js

```diff
@@ -0,0 +1,193 @@
+class DevicePreview extends HTMLElement {
```

Comment:
Below a certain width, the playground borders are getting covered up ([screenshot](https://i.gyazo.com/254a30a765f708b16f5a6fdf375ce6e4.png)). This doesn't happen on the [non-phone wrapper version](https://i.gyazo.com/383c6788d55ad5c7387fbc707ce45b9e.png).

Reply:
Great catch! Pushed up a fix (deploying), on < ~334px width screens the overflow will kick in and you can horizontally scroll within the container.

Reply:
Pushed up that change. It is fair to note that it is likely uncommon for a smaller mobile device to be using the doc demos through the playground. We may want to consider actually hiding the playground iframes in the future and have an action to launch the source in a "new tab", which would allow them to consume the example like a PWA.